deploy/helm-chart/values.yaml

```diff
@@ -0,0 +1,51 @@
+nginxGateway:
+  ## A GatewayClass resource with the name equal to the class must be deployed. Otherwise, NGINX Kubernetes Gateway
+  ## will fail to start. NGINX Kubernetes Gateway only processes resources that belong to its class - i.e. have the
+  ## "gatewayClassName" field resource equal to the class.
+  gatewayClass: "nginx"
+  image:
+    ## The NGINX Kubernetes Gateway image to use
+    repository: ghcr.io/nginxinc/nginx-kubernetes-gateway
+    tag: edge
+  imagePullPolicy: Always
+
+nginx:
+  ## The NGINX image to use
+  image:
+    repository: nginx
+    tag: "1.25"
+  imagePullPolicy: Always
+
+initContainer:
+  image:
+    ## The image the init container should use.
+    repository: busybox
+    tag: "1.36"
+
+serviceAccount:
+  annotations: {}
+  ## The name of the service account of the NGINX Kubernetes Gateway pods. Used for RBAC.
+  ## Autogenerated if not set or set to "".
+  # name: nginx-gateway
+
+service:
+  ## Creates a service to expose the NGINX Kubernetes Gateway pods.
+  create: true
+  ## The type of service to create for the NGINX Kubernetes Gateway.
+  type: LoadBalancer
+  ## The externalTrafficPolicy of the service. The value Local preserves the client source IP.
+  externalTrafficPolicy: Local
+  ## The annotations of the NGINX Kubernetes Gateway service.
+  annotations: {}
+
+  ## A list of ports to expose through the NGINX Kubernetes Gateway service. Follows the conventional Kubernetes yaml
```

Comment:
I suggest extending this comment:

A list of ports to expose through the NGINX Kubernetes Gateway service. Update it to match the listener ports from your Gateway resource. Follows the conventional Kubernetes yaml syntax for service ports.